packages/maskbook/src/plugins/Trader/types/trader.ts

```diff
@@ -29,6 +32,25 @@ export enum TradePool {
     Dodo,
 }
 
+export interface TradeComputed<T = unknown> {
+    strategy: TradeStrategy
+    inputToken: EtherTokenDetailed | ERC20TokenDetailed
+    outputToken: EtherTokenDetailed | ERC20TokenDetailed
+    inputAmount: BigNumber
+    outputAmount: BigNumber
+    estimatedInputAmount: BigNumber
+    estimatedOutputAmount: BigNumber
+    computed: {
+        maximumSold: BigNumber
+        minimumReceived: BigNumber
+        priceImpact: BigNumber
+        executionPrice: BigNumber
+        fee: BigNumber
+    }
+    path?: (EtherTokenDetailed | ERC20TokenDetailed)[]
+    trade_?: T
```

Comment:
Is it a typo or an internal property/

Reply:
Not a typo. This field refs to the original trade object for accessing unlisted stuff in the computed trade.